.github/workflows/prod-deployment.yaml

```
@@ -0,0 +1,82 @@
+jobs:
+  LMS-TAG-BASE-DEPLOYMENT-PROD:
+    name: Deployment
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Set TAG environment variable
+      run: echo "TAG=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:40: Double quote to prevent globbing and word splitting

(shellcheck)